### ISSUE TITLE
16 player version please

### ISSUE THREAD

**Issue by mirddes** [JeroenDeDauw/FinalRushPro5#23]
with permission i am happy to make the modifications myself and upload it to the vault.

**Comment by JeroenDeDauw** [comment]
Please make a pull request here and I will publish it. No need to duplicate the code and have the versions go out of sync. Let me know if you don't know how to do this.

**Comment by mirddes** [comment]
yep, ive got no clue what im doing. i only know how to add more slots to a map.

**Comment by JeroenDeDauw** [comment]
Ah, that's a pity. Git makes collaborating on things like this a lot easier.

You can just dump a zip of your version here and I'll see if I can integrate it. If you do make changes in the `.lua` files, please keep them minimal. In particular, do not reformat code. That would make integration very difficult.
